x-pack/plugins/triggers_actions_ui/public/application/sections/rule_details/components/rule_event_log_list_kpi.tsx

```diff
@@ -0,0 +1,251 @@
+  }
+);
+
+const ACTIONS_TOOLTIP = i18n.translate(
+  'xpack.triggersActionsUI.sections.ruleDetails.ruleEventLogListKpi.actionsTooltip',
+  {
+    defaultMessage: 'Action status of recent rule runs.',
+  }
+);
+
+const Stat = ({
+  title,
+  tooltip,
+  children,
+}: {
+  title: string;
+  tooltip: string;
+  children?: JSX.Element;
+}) => {
+  return (
+    <EuiPanel color="subdued">
+      <EuiFlexGroup justifyContent="spaceBetween">
+        <EuiFlexItem grow={false}>
+          <b>{title}</b>
+        </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <EuiIconTip content={tooltip} position="top" />
+        </EuiFlexItem>
+      </EuiFlexGroup>
+      <EuiSpacer />
+      {children}
+    </EuiPanel>
+  );
+};
+
+export type RuleEventLogListKPIProps = {
+  ruleId: string;
+  dateStart: string;
+  dateEnd: string;
+  outcomeFilter?: string[];
+  message?: string;
+  refreshToken?: number;
+} & Pick<RuleApis, 'loadExecutionKPIAggregations' | 'loadGlobalExecutionKPIAggregations'>;
+
+export const RuleEventLogListKPI = (props: RuleEventLogListKPIProps) => {
+  const {
+    ruleId,
+    dateStart,
+    dateEnd,
+    outcomeFilter,
+    message,
+    refreshToken,
+    loadExecutionKPIAggregations,
+    loadGlobalExecutionKPIAggregations,
+  } = props;
+  const {
+    notifications: { toasts },
+  } = useKibana().services;
+
+  const isInitialized = useRef(false);
+
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [kpi, setKpi] = useState<IExecutionKPIResult>();
+
+  const loadKPIFn = useMemo(() => {
+    if (ruleId === '*') {
+      return loadGlobalExecutionKPIAggregations;
+    }
+    return loadExecutionKPIAggregations;
+  }, [ruleId, loadExecutionKPIAggregations, loadGlobalExecutionKPIAggregations]);
+
+  const loadKPIs = async () => {
+    setIsLoading(true);
+    try {
+      const newKpi = await loadKPIFn({
+        id: ruleId,
+        dateStart: getParsedDate(dateStart),
+        dateEnd: getParsedDate(dateEnd),
+        outcomeFilter,
+        message,
+      });
+      setKpi(newKpi);
+    } catch (e) {
+      toasts.addDanger({
+        title: API_FAILED_MESSAGE,
+        text: e.body?.message ?? e,
+      });
+    }
+    setIsLoading(false);
+  };
+
+  useEffect(() => {
+    loadKPIs();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [ruleId, dateStart, dateEnd, outcomeFilter, message]);
+
+  useEffect(() => {
+    if (isInitialized.current) {
+      loadKPIs();
+    }
+    isInitialized.current = true;
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [refreshToken]);
+
+  if (isLoading || !kpi) {
+    return <CenterJustifiedSpinner />;
+  }
+
+  const getStatDescription = (element: React.ReactNode) => {
+    return (
+      <>
+        {element}
+        <EuiSpacer size="s" />
+      </>
+    );
+  };
+
+  return (
+    <EuiFlexGroup>
+      <EuiFlexItem grow={4}>
+        <Stat title="Response" tooltip={RESPONSE_TOOLTIP}>
```

Comment:
>        <Stat title="Response" tooltip={RESPONSE_TOOLTIP}>

Is the "Response" here equivalent to the "execution_status" in the "get rule" API results? i.e. is it the status of the latest run? If so, I think something like "Rule status" or "Latest run status" is clearer than "Response". Likewise, if the change is made in the stats, it should be also made in the "Response" column in the table.

Reply:
If this is a term that's used broadly, I can revisit this in a separate issue--no worries!